backend/models/schemas.py

```
@@ -141,3 +141,22 @@ def validate_report_type(cls, v):
             raise ValueError(f'Report type must be one of: {", ".join(allowed_types)}')
         return v
+
+class AppleAuthRequest(BaseModel):
+    idToken: str
+    userIdentifier: str
+    email: Optional[str] = None
+    firstName: Optional[str] = None
+    lastName: Optional[str] = None
+    
+    @validator('idToken')
+    def validate_id_token(cls, v):
```

Pylint (3.3.7)

[error] Method 'validate_id_token' should have "self" as first argument

(E0213)